ec2/sc-ec2-linux-jumpcloud.yaml

```diff
@@ -386,14 +386,15 @@ Resources:
             01_tag_root_disk:
               command: !Join
                 - ''
-                - - "EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id) "
+                - - "set -x;"
+                  - "EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id) &&"
```

Comment:
`&&` doesn't seem necessary here?  at least it worked as is here: https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/managed-ec2-linux-v1.j2#L220

Reply:
This script is working as is

Reply:
All changes made have been tested and were necessary